notebooks/enterprise-search/elastic-crawler-to-open-crawler-migration.ipynb

```diff
@@ -0,0 +1,555 @@
+    "    if not existing_schedule_value:\n",
+    "        # query ES to get this Crawler's specific time schedule\n",
+    "        schedules_result = es_client.search(\n",
+    "            index=\".elastic-connectors-v1\",\n",
+    "            query={\"match\": {\"index_name\": output_index}},\n",
+    "            _source=[\"index_name\", \"scheduling\"],\n",
+    "        )\n",
+    "        # update schedule field with cron expression if specific time scheduling is enabled\n",
+    "        if schedules_result[\"hits\"][\"hits\"][0][\"_source\"][\"scheduling\"][\"full\"][\n",
+    "            \"enabled\"\n",
+    "        ]:\n",
+    "            quartz_schedule = schedules_result[\"hits\"][\"hits\"][0][\"_source\"][\n",
+    "                \"scheduling\"\n",
+    "            ][\"full\"][\"interval\"]\n",
+    "            crawler_config[\"schedule\"] = convert_quartz_to_cron(quartz_schedule)\n",
+    "            print(\n",
+    "                f\"{crawler_counter}.) Crawler {output_index} has the schedule {crawler_config['schedule']}\"\n",
+    "            )\n",
+    "            crawler_counter += 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b1586df2-283d-435f-9b08-ba9fad3a7e0a",
+   "metadata": {},
+   "source": [
+    "### Step 5: Creating the Open Crawler YAML configuration files\n",
+    "\n",
+    "In this final step, we will be creating the actual YAML files you need to get up and running with Open Crawler!\n",
+    "\n",
+    "The upcoming cell performs some final transformations to the in-memory data structure that is keeping track of your configurations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dd70f102-33ee-4106-8861-0aa0f9a223a1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Final transform of the in-memory data structure to a form we can dump to YAML\n",
+    "# for each crawler, collect all of its domain configurations into a list\n",
+    "for crawler_oid, crawler_config in inflight_configuration_data.items():\n",
+    "    all_crawler_domains = []\n",
+    "\n",
+    "    for domain_config in crawler_config[\"domains_temp\"].values():\n",
+    "        all_crawler_domains.append(domain_config)\n",
+    "    # create a new key called \"domains\" that points to a list of domain configs only - no domain_oid values as keys\n",
+    "    crawler_config[\"domains\"] = all_crawler_domains\n",
+    "    # delete the temporary domain key\n",
+    "    del crawler_config[\"domains_temp\"]\n",
+    "    print(f\"Transform for {crawler_oid} complete!\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e611a486-e12f-4951-ab95-ca54241a7a06",
+   "metadata": {},
+   "source": [
+    "#### **Wait! Before we continue onto creating our YAML files, we're going to need your input on a few things.**\n",
+    "\n",
+    "In the following cell, please enter the following details about the _Elasticsearch instance you will be using with Open Crawler_:\n",
```

Comment:
Maybe we could be more explicit that the instance can be local or running on Elastic Cloud Hosted/Serverless?